test-scripts/local-is-deployment/DeploymentPatterns/pattern-2/deploy.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+# ----------------------------------------------------------------------------
+#
+# Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
+# WSO2 Inc. licenses this file to you under the Apache License,
+# Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+# ----------------------------------------------------------------------------
+
+# This script depends on curl and unzip, so make sure it is already installed. 
+
+echo "This is a Dummy Deployment creation scripts to deploy"
+echo "Here the test infrastructure is your local machine.."
+pwd
+#if you do not have a account, please create a account at https://wso2.com/
+WOS2_USER=yasassri@wso2.com
+WSO2_PASSWORD=xxxxxxx
+DEPLOYMENT_EP_FILE_NAME=deployment_eps
+
+echo "Downloading the IS 5.4.0 distribution"
+#curl -k https://product-dist.wso2.com/products/identity-server/5.4.0/wso2is-5.4.0.zip --user $WSO2_USER:$WSO2_PASSWORD -o is.zip
```

Comment:
curl command is commented out here. 

Shall we ask the users to place the IS distribution in the same place as this file is? 
Then, we can proceed if the file is present. Execution should stop with an error if the file is missing.

Reply:
Since we are planning to use these scripts for day-to-day testing, waiting for a download is a waste of time. Hence, better to ask the users to place the distribution in the current directory.